### PR TITLE
[Feature] article unread

### PR DESCRIPTION
**Pull request by wildskyf**
mark article unread after created

### CODE REVIEW

File: src/reducers/utils.js [hunk 1]
```diff
@@ -160,6 +160,10 @@ export const reduceUpdateData = (state, action) => {
   return state.mergeDeepIn([myId], data)
 }
 
+export const isNullTimeStamp = timestamp => {
```
Comment (sammui-AILabs) [inline src/reducers/utils.js]: Does `{"T":0,"NT":0}` happen?

If so, need to confirm with @chhsiao1981 as it should always return value.

File: src/reducers/BoardPage.js [hunk 2]
```diff
@@ -456,9 +460,9 @@ const postprocessGetMoreArticles = (myId, result, usersInfo) => {
       NPush:            each.NPush,
       Title:            each.Title,
       CreateTS:         each.CreateTS ? each.CreateTS : utils.emptyTimeStamp(),
-      UpdateTS:         each.UpdateTS ? each.UpdateTS : utils.emptyTimeStamp(),
+      UpdateTS:         each.UpdateTS ? each.UpdateTS : createTS,
       LastSeen:         each.L ? each.L : utils.emptyTimeStamp(),
-      CommentCreateTS:  each.c ? each.c : utils.emptyTimeStamp(),
+      CommentCreateTS:  each.c && !utils.isNullTimeStamp(each.c) ? each.c : createTS,
```
Comment (sammui-AILabs) [inline src/reducers/BoardPage.js]: should be `each.c && !utils.isNullTimeStamp(each.c) ? each.c : UpdateTS,` instead?

File: src/reducers/BoardPage.js [hunk 3]
```diff
@@ -339,14 +341,14 @@ const postprocessGetArticleList = (myId, result, isFirstFetch, usersInfo) => {
       NBoo:             each.NB,
       NPush:            each.NP,
       Title:            each.Title,
-      CreateTS:         each.CreateTS ? each.CreateTS : utils.emptyTimeStamp(),
-      UpdateTS:         each.UpdateTS ? each.UpdateTS : utils.emptyTimeStamp(),
+      CreateTS:         createTS,
+      UpdateTS:         each.UpdateTS ? each.UpdateTS : createTS,
       LastSeen:         each.L ? each.L : utils.emptyTimeStamp(),
-      CommentCreateTS:  each.c ? each.c : utils.emptyTimeStamp(),
+      CommentCreateTS:  each.c && !utils.isNullTimeStamp(each.c) ? each.c : createTS,
```
Comment (sammui-AILabs) [inline src/reducers/BoardPage.js]: should be `each.c && !utils.isNullTimeStamp(each.c) ? each.c : UpdateTS,` instead?

File: src/reducers/utils.js (hunk 1 above)
Comment (chhsiao1981) [reply src/reducers/utils.js]: It's possible in LastSeen

File: src/reducers/BoardPage.js (hunk 2 above)
Comment (chhsiao1981) [reply src/reducers/BoardPage.js]: It's also possible when there is no comments.

Comment (sammui-AILabs) [reply src/reducers/BoardPage.js]: Got it, thanks for the confirmation.

File: src/reducers/utils.js (hunk 1 above)
Comment (sammui-AILabs) [reply src/reducers/utils.js]: This is for CommentCreateTS, but got your answer in another comment:

Comment (sammui-AILabs) [reply src/reducers/utils.js]: `It's also possible when there is no comments.`

File: src/reducers/BoardPage.js (hunk 3 above)
Comment (wildskyf) [reply src/reducers/BoardPage.js]: I don't have preference, so I just changed it to `updateTS` as you thought ;)

**Pull request merged**